### ISSUE TITLE
How to disable border on DataGrid?

### ISSUE THREAD

**Issue by shtse8** [TomaszRewak/C-sharp-console-multiplexer#23]
Thanks for making this amazing framework of making console GUI.

I don't know how to disable the border line between rows in DataGrid.

**Comment by TomaszRewak** [comment]
Thanks!

The current implementation doesn't allow for that: https://github.com/TomaszRewak/C-sharp-console-gui-framework/blob/master/ConsoleGUI/Controls/DataGrid.cs#L88
The border style is hardcoded in the `DataGrid` `Cell` getter. But that should be easy enough to change.
What result are you aiming for? Do you want to get rid of those vertical lines completely (so that all lines are displayed one after the other), or do you want to replace them with empty cells by overwriting the border style?

**Comment by shtse8** [comment]
I want to remove all horizontal and vertical line inside the grid. just leaving the outline. especially the horizontal line, it used up so much space of the screen.
Also, how can I change the aligement of the text inside  each cell?

**Comment by TomaszRewak** [comment]
So for both of those there will be some dev work required.
I think that I should be able to find some time for that this week.

**Comment by TomaszRewak** [comment]
I've made the border style and the text alignment customizable with the 7cd54b6bdd8a05b882f8cdafd487d0a5878baa6d.
It's already released in v1.4.0.

Please see this example:
```
var leaderboard = new DataGrid<Player>
{
	Columns = new[]
	{
		new DataGrid<Player>.ColumnDefinition("Name", 10, p => p.Name, foreground: p => p.Name == "Tomasz" ? (Color?)new Color(100, 100, 220) : null),
		new DataGrid<Player>.ColumnDefinition("Surname", 10, p => p.Surname),
		new DataGrid<Player>.ColumnDefinition("Birth date", 15, p => p.BirthDate.ToShortDateString(), textAlignment: TextAlignment.Center),
		new DataGrid<Player>.ColumnDefinition("Points", 5, p => p.Points.ToString(), background: p => p.Points > 20 ? (Color?)new Color(0, 220, 0) : null, textAlignment: TextAlignment.Right)
	},
	Data = new[]
	{
		new Player("John", "Connor", new DateTime(1985, 2, 28), 10),
		new Player("Ellen", "Ripley", new DateTime(2092, 1, 1), 23),
		new Player("Jan", "Kowalski", new DateTime(1990, 4, 10), 50),
		new Player("Tomasz", "Rewak", new DateTime(1900, 1, 1), 0),
	},
	Style = DataGridStyle.NoHorizontalCellBorders
};
```

![image](https://user-images.githubusercontent.com/5797540/117585139-0ddbc880-b111-11eb-9bf0-c2940e4888e0.png)
